osh/hub/scripts/osh-retention

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
+
+"""
+Script for cron that performs the retention policy
+"""
+import datetime
+import logging
+import os
+import shutil
+from datetime import date
+
+# Set up django configuration
+import django
+from kobo.client.constants import FINISHED_STATES
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'osh.hub.settings'
+django.setup()
+from kobo.hub.models import Task  # noqa
+
+from osh.hub.scan.models import RetentionPolicySetting  # noqa
+from osh.hub.scan.models import TaskResultsRemoval  # noqa
+
+logger = logging.getLogger("osh.hub.scripts.osh-retention")
```

Comment:
Is there a reason why we don't use `__name__` here as we do in the rest of the project?
```suggestion
logger = logging.getLogger(__name__)
```

Reply:
I used that instead of `__name__` because at the time of logging, logging was not done correctly and there was no output. At the time of using that, everything was fixed. I also saw some other modules use a separate logging, such as `logger = logging.getLogger('osh_api_client')`

Reply:
@jperezdealgaba `osh/hub/scripts/osh-xmlrpc-client.py` is a debugging script that we do not install, so it might have some technical debt.

Reply:
I have thinking and trying about this. The rest of the files that uses `__name__` doesn't have a main function as the osh-retention does. even the `osh-stats` uses another module, that makes that using `__name__` will resolve in the correct file. `osh-retention` uses one file. I can separate the modules and create two, but I don't think here it is necessary.